packages/worker/src/worker.ts

```diff
@@ -543,17 +543,26 @@ export class Worker {
     logger: Logger
   ): Promise<WorkflowBundleWithSourceMap | undefined> {
     if (compiledOptions.workflowsPath) {
+      if (compiledOptions.workflowBundle) {
+        logger.info('WorkerOptions.workflowsPath is set, so workflowBundle option is ignored');
```

Comment:
This should throw an error instead of logging in info level which is often disabled